org.eclipse.jdt.core.compiler.batch/src/org/eclipse/jdt/internal/compiler/ast/InstanceOfExpression.java

```diff
@@ -39,9 +39,9 @@ public class InstanceOfExpression extends OperatorExpression {
 	public TypeReference type;
 	public LocalDeclaration elementVariable;
 	public Pattern pattern;
-	static final char[] SECRET_INSTANCEOF_PATTERN_EXPRESSION_VALUE = " instanceOfPatternExpressionValue".toCharArray(); //$NON-NLS-1$
+	static final char[] SECRET_EXPRESSION_VALUE = " secretExpressionValue".toCharArray(); //$NON-NLS-1$
 
-	public LocalVariableBinding secretInstanceOfPatternExpressionValue = null;
+	private LocalVariableBinding secretExpressionValue = null;
 
```

Comment:
This is a cosmetic change that significantly shortens the identifiers/names without sacrificing readability/maintainability